### PR TITLE
docs: add link and image of jdk 8 install to build.md 

### LINKED ISSUE #1803: docs: update README.md and BUILD.md files
### Description
As a person interested in using and contributing to the Cactus project, I want the documentations to be updated in order to reflect all the steps needed to set up the project on my local machine. Although the `BUILD.md` file shows most of what is needed to set up Cactus, it misses a step of installing `jdk8` onto the user's local machine (the video is up on the Hyperledger Cactus playlist, it just wasn't added to the document itself). 

Furthermore, there could be some more cleaning up of documentation (grammar, formatting, etc...) that can help the user have a better time learning about Cactus. Our current documentation is not up to date on the number of `connectors` available on Cactus.  

### Acceptance Criteria
1. Update the `BUILD.md`to add the step of installing `jdk8`
2. Add current `connectors` to the`README.md` page and link the package's `README.md` page for easy acess
3. Clean up overall grammar and formatting on other `.md` files avaliable

### PR DESCRIPTION
**Pull request by Leeyoungone**
Fixes #1803

Signed-off-by: Youngone Lee <youngone.lee@accenture.com>

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ This is a video guide to setup Hyperledger Cactus on your local machine.
 
 [![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/94xoV9Vfu14/0.jpg)](https://www.youtube.com/watch?v=94xoV9Vfu14)
 
+
+### Installing jkd8
```
Comment (jagpreetsinghsasan) [inline BUILD.md:30]: `jdk8` a typo

**Review by jagpreetsinghsasan — CHANGES_REQUESTED** [review]

File: BUILD.md (hunk 1 above)
Comment (Leeyoungone) [reply BUILD.md:30]: I managed to make a typo for a four character word lol, thank you for checking :)

Comment (petermetz) [reply BUILD.md:30]: :smile:

**Review by petermetz — CHANGES_REQUESTED** [review]
@Leeyoungone The issue it is fixing looked good in terms of the scope/description of it but somehow the PR morphed that into a big update on the whitepaper itself instead of the specific README files of the connectors. So what you are doing is legit, but I think the content could be moved from the whitepaper.md file to the README.md files of the relevant connectors such as `packages/cactus-plugin-ledger-connector-corda/README.md` etc.

**Comment by Leeyoungone** [comment]
@petermetz Thanks for the comment ! Hope you're well :) 

> PR morphed that into a big update on the whitepaper itself instead of the specific README files of the connectors

Oops, I'm sorry! I misnamed this PR. The original issue I had up for this was to update both of the README and the whitepaper for the connector and the video.  

> content could be moved from the whitepaper.md file to the README.md files of the relevant connectors such as `packages/cactus-plugin-ledger-connector-corda/README.md` etc

The individual `plugin-ledger-connector`'s README.md files all already contain this information. However, the format of our current whitepaper.md file contains bullet points of what each connector within Cactus does. I've updated the whitepaper to reflect this format (providing a little bit of information for each connector within whitepaper.md) but if you think this section is no longer necessary, I can definitely take it off!

File: BUILD.md (hunk 1 above)
Comment (petermetz) [reply BUILD.md:30]: @Leeyoungone It still has the typo in there AFAICT. (Also, sorry for the slow response/review here)

**Review by petermetz — CHANGES_REQUESTED** [review]

**Comment by petermetz** [comment]
> Oops, I'm sorry! I misnamed this PR. The original issue I had up for this was to update both of the README and the whitepaper for the connector and the video.

@Leeyoungone No worries, just please make sure to name it something that describes a little distinctly what is being changed because "updating file X and Y" can accurately describe 99% of the pull requests of all time so after the fact if someone is reading just that text and trying to figure out if it's an interesting change to them or not they'll have no choice but to go deeper and inspect the diff directly. :-)

If you like, it's also totally cool to break this into two PRs where one is about the video change in BUILD.md and the other one is about the whitepaper change

**Comment by Leeyoungone** [comment]
> make sure to name it something that describes a little distinctly what is being changed because "updating file X and Y" can accurately describe 99% of the pull requests

Sounds good! I appreciate you helping me on honing good developer skills :)) 


> If you like, it's also totally cool to break this into two PRs where one is about the video change in BUILD.md and the other one is about the whitepaper change

I can definitely do that! sorry for delay

**Review by petermetz — APPROVED** [review]
@Leeyoungone 

> Sounds good! I appreciate you helping me on honing good developer skills :))

My pleasure, I'm glad it's helpful info!

> I can definitely do that! sorry for delay

No worries, the reviews were slow on this one as well, we'll get there.

LGTM

**Review by jagpreetsinghsasan — APPROVED** [review]
LGTM

**Pull request merged**